modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,137 @@
+// Module included in the following assemblies:
+//
+// * understanding_openshift_gitops/about-redhat-openshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-openshift-go-common-terms_{context}"]
+= Glossary of common terms for OpenShift {gitops-shortname}
+
+This glossary defines common OpenShift {gitops-shortname} terms.
+
+Argo CD::
+An open-source declarative tool that automates the continuous deployment (CD) of Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+{gitops-shortname}::
+{gitops-shortname} is a declarative way to implement continuous deployment for cloud native applications. In {gitops-shortname}, a Git repository contains deployment resources, which Argo CD then keeps synchronizing with its cluster state.
+
+{gitops-title}::
+ An Operator that uses Argo CD as the declarative {gitops-shortname} engine for enabling {gitops-shortname} workflows across multicluster OpenShift and Kubernetes infrastructure.
+
+{gitops-shortname} CLI ({gitops-shortname} `argocd` CLI)::
+A tool to configure and manage {gitops-title} and Argo CD resources from the command line. 
+
+Control plane ({gitops-shortname} control plane)::
+In the {gitops-shortname} context, control plane is the `openshift-gitops` namespace, where, by default, the {gitops-title} installs an instance of Argo CD. This control plane allows you to provision, manage, and operate Argo CD across networks, instances, and clusters. Examples include a control plane repository wherein you can create one or more application custom resources and express an intent to deploy and sync Kubernetes manifests from Git.
```

Comment:
Question: In the second part of the sentence, should from Git -> from a Git repository?

Reply:
not relevant anymore as the concerned content part is removed.